service/src/main/java/bio/terra/cbas/dependencies/wes/CromwellService.java

```diff
@@ -83,18 +83,27 @@ public WorkflowQueryResult runSummary(String runId) throws ApiException {
 
   public Object getOutputs(String id)
       throws ApiException, DependencyNotAvailableException, AzureAccessTokenException {
-    return cromwellClient.wesAPI().getRunLog(id).getOutputs();
+    ApiClient client = cromwellClient.getWriteApiClient();
+
+    return cromwellClient.wesAPI(client).getRunLog(id).getOutputs();
   }
 
-  public WorkflowDescription describeWorkflow(String workflowUrl) throws ApiException {
-    return cromwellConfig.womtoolApi().describe(API_VERSION, null, workflowUrl, null, null, null);
+  public WorkflowDescription describeWorkflow(String workflowUrl)
+      throws ApiException, DependencyNotAvailableException, AzureAccessTokenException {
+    ApiClient client = cromwellClient.getWriteApiClient();
```

Comment:
same here - should this be available to everyone? (I have a feeling I'm missing something, but I figure I ought to ask!)

Reply:
This one I'm on the fence about. @salonishah11 had a good reason for why this should be a write function over a read function. Saloni, do you remember what your thoughts were?

Reply:
The `describeWorkflow` endpoint is used in 2 places currently, one when a new method is being created and other when submitting a workflow. My initial thoughts were that it would make sense to call the individual Cromwell Runner / Writer especially for when submitting a workflow since the submission will be made to user's Cromwell rather than the Cromwell Reader.

Reply:
And I'm assuming that we don't want a user with read access to be able to create new methods?

Reply:
Checking of read/write access will already happen before this endpoint is called. Here we just need to figure out whether we should call the user's Cromwell (which is Cromwell Runner) or the single reader/summarizer Cromwell that is created with Workflows app.

Reply:
I also like this being a read action, for most of the same reasons - 
1. The reader should be much faster
2. The action will probably be called separately from submitting a workflow, so CBAS might not even have started the runner instance yet
3. Imagine a user importing a workflow for someone else to run. It doesn't make sense that the importing user has to start up a Cromwell instance just to run `describe`, then shut it down and have the second user start their runner to do the running